pkg/sensors/base/base_linux.go

```diff
@@ -33,6 +41,35 @@ func GetDefaultMaps() []*program.Map {
 		MatchBinariesGenMap,
 		ErrMetricsMap,
 	}
+	// The BPF ring buffer is available from v5.8, but rather than add another set of
```

Comment:
qq does this leave kernels `5.8>=`, `<5.11` unsupported? not sure it matters much just curious